mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java

```diff
@@ -203,10 +203,7 @@ protected Boolean doInBackground(String... strings) {
                         Log.d(TAG, "Socket connected");
                         s.close();
                         return true;
-                    } catch (MalformedURLException e) {
-                        Log.e(TAG, e.getMessage());
-                        return false;
-                    } catch (IOException e) {
+                    } catch (Exception e) {
```

Comment:
I don't know how to do the Signed-off-by part and cannot do it in next 2 days because I cannot access my dev machine.